### PR TITLE
V9b

### PR DESCRIPTION
**Pull request by paulb777**
Merging breaking change v9b branch into master in advance of Firebase 9.0.0 release.

#no-changelog

### CODE REVIEW

**Comment by ncooke3** [comment]
kicked off re-runs of two jobs that looked like flakes: 
- `watchos-sample / tv-sample-build-test`
- `crashlytics / pod-lib-lint (tvos)`

File: FirebaseCoreInternal.podspec [hunk 1]
```diff
@@ -0,0 +1,74 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseCoreInternal'
+  s.version          = '8.13.0'
```
Comment (ncooke3) [inline FirebaseCoreInternal.podspec:3]:
```suggestion
  s.version          = '8.15.0'
```

File: FirebaseCoreExtension.podspec [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    s.ios.deployment_target = '9.0'
+    s.osx.deployment_target = '10.12'
+    s.tvos.deployment_target = '10.0'
+    s.watchos.deployment_target = '6.0'
+
+    s.source_files = 'FirebaseCore/Extension/*.[hm]'
+    s.public_header_files = 'FirebaseCore/Extension/*.h'
+
+    s.dependency 'FirebaseCore', '~> 8.12'
```
Comment (ncooke3) [inline FirebaseCoreExtension.podspec:31]:
```suggestion
    s.dependency 'FirebaseCore', '~> 8.15'
```

**Review by ncooke3 — APPROVED** [review]

**Comment by ncooke3** [comment]
I dont think any of my comments are blocking this merge.

File: .github/workflows/functions.yml [hunk 3]
```diff
@@ -86,60 +81,45 @@ jobs:
...
+  #           scripts/third_party/travis/retry.sh scripts/test_quickstart.sh Functions true)
+  #   - name: Test swift quickstart
+  #     run: ([ -z $plist_secret ] ||
+  #           scripts/third_party/travis/retry.sh scripts/test_quickstart.sh Functions true swift)
 
   functions-cron-only:
     # Don't run on private repo.
-    if: github.event_name == 'schedule' && github.repository == 'Firebase/firebase-ios-sdk'
+# TODO: Uncomment below
+#    if: github.event_name == 'schedule' && github.repository == 'Firebase/firebase-ios-sdk'
```
[the remark this answers is not in the document]
Comment (paulb777) [reply .github/workflows/functions.yml:115]: Let's get 9.0.0 podspecs published to SpecsDev first and then reenable these

File: FirebaseCoreInternal.podspec (hunk 1 above)
Comment (paulb777) [reply FirebaseCoreInternal.podspec:3]: These are kind of wrong either way and will go to 9.0.0 shortly

**Review by ryanwilson — APPROVED** [review]

**Pull request merged**